package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "google-listings-and-ads",
-	"version": "2.6.3",
```

Comment:
Is this commit necessary?

Reply:
I think I did not add this commit. But I merged the latest base branch and solved the conflicts so the `package-lock.json` is the latest version now.